src/frontend/screens/Onboarding/walletSelection/screens/import/index.tsx

```diff
@@ -61,26 +65,30 @@ const ImportScreen = ({
           `By importing, your MetaMask installation and  settings will be imported into HyperPlay.`
         )}
       </div>
-      {browserSelected === '' ? (
+      {browserSelected === null ||
+      importOptions[browserSelected] === undefined ? (
```

Comment:
I don't understand this condition `importOptions[browserSelected] === undefined`

Reply:
I agree, I think all these conditions pollute the code and make it less readable. Perhaps some things could be done on a helper function with clear statements so it is easier for us to maintain this code in the future.

Reply:
sure I can refactor and clean this component up. Was trying to change as little as possible to get this out. The git diff makes it look like I changed/wrote more of this than I did